modnet/preprocessing.py

```diff
@@ -157,7 +158,7 @@ def get_cross_nmi(df_feat: pd.DataFrame, **kwargs) -> pd.DataFrame:
             mutual_info.loc[y_feat, x_feat] = mutual_info.loc[x_feat, y_feat] = I_xy
 
     mutual_info.fillna(0, inplace=True) # if na => no relation => set to zero
-    return mutual_info
+    return mutual_info, diag # diag can be useful for future elimination based on entropy without the need of recomputing the cross NMI
```

Comment:
Could we just set the diagonal values in the dataframe instead of setting them to 1, so we don't need to return this? This change breaks the existing API otherwise